<!DOCTYPE html>
<html lang="en">
    <head>
        <title>Mélina Gallopin</title>
        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
        <meta name="author" content="owwwlab.com">
        <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
        
        <meta name="description" content="A theme for faculty profile page" />
        <meta name="keywords" content="faculty profile, theme,css, html, jquery, transition, transform, 3d, css3" />

        <link rel="shortcut icon" href="../favicon.ico">

        <!--CSS styles-->
        <link rel="stylesheet" href="css/bootstrap.css">
        <link rel="stylesheet" href="css/font-awesome.min.css">  
        <link rel="stylesheet" href="css/perfect-scrollbar-0.4.5.min.css">
        <link rel="stylesheet" href="css/magnific-popup.css">
        <link rel="stylesheet" href="css/style.css">
        <link id="theme-style" rel="stylesheet" href="css/styles/default.css">

        
        <!--/CSS styles-->
        <!--Javascript files-->
        <script type="text/javascript" src="js/jquery-1.11.3.min.js"></script>
        <script type="text/javascript" src="js/TweenMax.min.js"></script>
        <script type="text/javascript" src="js/jquery.touchSwipe.min.js"></script>
        <script type="text/javascript" src="js/jquery.carouFredSel-6.2.1-packed.js"></script>
        
        <script type="text/javascript" src="js/modernizr.custom.63321.js"></script>
        <script type="text/javascript" src="js/jquery.dropdownit.js"></script>

        <script type="text/javascript" src="js/ScrollToPlugin.min.js"></script>

        <script type="text/javascript" src="js/bootstrap.min.js"></script>

        <script type="text/javascript" src="js/jquery.mixitup.min.js"></script>

        <script type="text/javascript" src="js/masonry.min.js"></script>

        <script type="text/javascript" src="js/perfect-scrollbar-0.4.5.with-mousewheel.min.js"></script>
        <script type="text/javascript" src="js/jquery.nicescroll.min.js"></script>

        <script type="text/javascript" src="js/magnific-popup.js"></script>
        <script type="text/javascript" src="js/custom.js"></script>

        <!--/Javascript files-->

    </head>
    <body>
        <!-- Google Tag Manager -->
        <noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M6HHNF"
        height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
        <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
        new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
        j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
        '//www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
        })(window,document,'script','dataLayer','GTM-M6HHNF');</script>
        <!-- End Google Tag Manager -->

        <div id="wrapper">
            <a href="#sidebar" class="mobilemenu"><i class="fa fa-reorder"></i></a>

            <div id="sidebar">
                <div id="sidebar-wrapper">
                    <div id="sidebar-inner">
                        <!-- Profile/logo section-->
                        <div id="profile" class="clearfix">
                            <div class="portrate hidden-xs"></div>
                            <div class="title">
                                <h2>Mélina Gallopin</h2>
                                <!--h3>Stanford University</h3-->
                            </div>   
                        </div>
                        <div class="flags">
                            <a href="/publication.html"><img src="img/fr.png"></a>
                            <a href="en/publication.html"><img src="img/en.png"></a>
                        </div>

                        <!-- /Profile/logo section-->

                        <!-- Main navigation-->
                        <div id="main-nav">
                            <ul id="navigation">
                                <li>
                                  <a href="index.html">
                                    <i class="fa fa-user"></i>
                                    <div class="text">Situation actuelle</div>
                                  </a>
                                </li>  
                                
                                 <li>
                                  <a href="research.html">
                                    <i class="fa fa-book"></i>
                                    <div class="text">Recherche</div>
                                  </a>
                                </li> 
                                
                                <li class="currentmenu">
                                  <a href="publication.html">
                                    <i class="fa fa-edit"></i>
                                    <div class="text">Publications</div>
                                  </a>
                                </li> 
                                <li>
                                  <a href="logiciels.html">
                                    <i class="fa fa-linux"></i>
                                    <div class="text">Logiciels</div>
                                  </a>
                                </li> 
                                <li>
                                  <a href="teaching.html">
                                    <i class="fa fa-clock-o"></i>
                                    <div class="text">Enseignement</div>
                                  </a>
                                </li>

          <!--                       <li>
                                  <a href="gallery.html">
                                    <i class="fa fa-picture-o"></i>
                                    <div class="text">Autres</div>
                                  </a>
                                </li> -->

                                <li>
                                  <a href="contact.html">
                                      <i class="fa fa-calendar"></i>
                                      <div class="text">Me contacter</div>
                                  </a>
                                </li>

          <!--                       <li class="external">
                                  <a href="cv.pdf">
                                      <i class="fa fa-download"></i>
                                      <div class="text">Download CV</div>
                                  </a>
                                </li> -->
                            </ul>
                        </div>
                        <!-- /Main navigation-->
                        <!-- Sidebar footer -->
                        <div id="sidebar-footer">
                            <div class="social-icons">
                                <ul>
                                    <!-- <li><a href="#"><i class="fa fa-facebook"></i></a></li> -->
                                    <li><a href="https://twitter.com/melinagallopin"><i class="fa fa-twitter"></i></a></li>
                                    <li><a href="https://www.linkedin.com/in/m%C3%A9lina-gallopin-53541420"><i class="fa fa-linkedin"></i></a></li>
                                    <li><a href="https://github.com/Gallopin"><i class="fa fa-github"></i></a></li>
                                </ul>
                            </div>


        
                             <div id="copyright">Copyright Mélina Gallopin</div>
                    
                        </div>
                        <!-- /Sidebar footer -->
                    </div>

                </div>
            </div>

            <div id="main">
            
                <div id="publications" class="page">
                    <div class="page-container">
                        <div class="pageheader">
                            <div class="headercontent">
                                <div class="section-container">
                                    
                                    <h2 class="title">Publications</h2>
                                    <div class="row">
                                        <div class="col-md-12">
                                            <p> </p>
                                        </div>
                                    </div>
                                    
                                </div>
                            </div>
                        </div>

                        <div class="pagecontents">

                            <div class="section color-2" id="pub-grid">
                                <div class="section-container">
                                    
                                    <div class="row">
                                        <div class="col-md-12">
                                            <div class="pitems">
                                                <div class="item mix cpaper" data-year="2017">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                            <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1372-2" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>
                                                            <a href="https://genomebiology.biomedcentral.com/track/pdf/10.1186/s13059-017-1372-2?site=genomebiology.biomedcentral.com" class="tooltips" title="Download" target="_blank">
                                                                <i class="fa fa-cloud-download"></i>
                                                            </a>
                                                            
                                                        </div>


                                                        <h4 class="pubtitle">Exhaustive capture of biological variation in RNA-seq data through k-mer decomposition.</h4>
                                                        <div class="pubauthor">  Audoux J., Philippe N., Chikhi R., Salson M., <strong>Gallopin M.</strong>, Gabriel M., Le Coz J., Drouineau E., Commes T., Gautheret D. </div>
                                                        <div class="pubcite"><span class="label label-success">Publication</span> Genome Biology, 2017, 18:243. </div>
                                                        
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>We introduce a k-mer-based computational protocol, DE-kupl, for capturing local RNA variation in a set of RNA-seq libraries, independently of a reference genome or transcriptome. DE-kupl extracts all k-mers with differential abundance directly from the raw data files. This enables the retrieval of virtually all variation lying in an RNA-seq dataset. This variation is subsequently assigned to biological events such as differential lncRNAs, splice and polyadenylation variants, introns, repeats, editing/mutation events, or exogenous RNA. Applying DE-kupl to different human RNA-seq datasets identified multiple types of novel events, reproducibly across independent RNA-seq experiments.
                                                        </p>
                                                    </div>
                                                </div>
                                                <div class="item mix cpaper" data-year="2017">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                            <a href="https://arxiv.org/abs/1701.07899" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>
                                                            <a href="https://arxiv.org/pdf/1701.07899.pdf" class="tooltips" title="Download" target="_blank">
                                                                <i class="fa fa-cloud-download"></i>
                                                            </a>
                                                            
                                                        </div>


                                                        <h4 class="pubtitle">Nonlinear network-based quantitative trait prediction from transcriptomic data.</h4>
                                                        <div class="pubauthor"> Devijver E., <strong>Gallopin M.</strong>, Perthame E.</div>
                                                        <div class="pubcite"><span class="label label-warning">Pré-publication</span> Soumis, 2017.</div>
                                                        
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>Quantitatively predicting phenotype variables by the expression changes in a set of candidate genes is of great interest in molecular biology but it is also a challenging task for several reasons. First, the collected biological observations might be heterogeneous and correspond to different biological mechanisms. Secondly, the gene expression variables used to predict the phenotype are potentially highly correlated since genes interact though unknown regulatory networks. In this paper, we present a novel approach designed to predict quantitative trait from transcriptomic data, taking into account the heterogeneity in biological samples and the hidden gene regulatory networks underlying different biological mechanisms. The proposed model performs well on prediction but it is also fully parametric, which facilitates the downstream biological interpretation. The model provides clusters of individuals based on the relation between gene expression data and the phenotype, and also leads to infer a gene regulatory network specific for each cluster of individuals. We perform numerical simulations to demonstrate that our model is competitive with other prediction models, and we demonstrate the predictive performance and the interpretability of our model to predict alcohol sensitivity from transcriptomic data on real data from Drosophila Melanogaster Genetic Reference Panel (DGRP).</p>
                                                    </div>
                                                </div>
                                                <div class="item mix jpaper" data-year="2016">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                            <a href="http://amstat.tandfonline.com/doi/abs/10.1080/01621459.2016.1247002#.WfXeIBNSzVo" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>
                                                            <!--a href="http://faculty-gsb.stanford.edu/aaker/pages/documents/CultivatingAdmirationinBrands_JCP2012.pdf" class="tooltips" title="Download" target="_blank">
                                                                <i class="fa fa-cloud-download"></i>
                                                            </a-->
                                                            
                                                        </div>


                                                        <h4 class="pubtitle">Block-diagonal covariance selection for high-dimensional Gaussian graphical model.</h4>
                                                        <div class="pubauthor">Devijver E., <strong>Gallopin M.</strong>.</div>
                                                        <div class="pubcite"><span class="label label-success">Publication</span> Journal of the American Statistical Association, 2016, DOI: 10.1080/01621459.2016.1247002.</div>
                                                        
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>Gaussian graphical models are widely used to infer and visualize networks of dependencies between continuous variables. However, inferring the graph is difficult when the sample size is small compared to the number of variables. To reduce the number of parameters to estimate in the model, we propose a nonasymptotic model selection procedure supported by strong theoretical guarantees based on an oracle type inequality and a minimax lower bound. The covariance matrix of the model is approximated by a block-diagonal matrix. The structure of this matrix is detected by thresholding the sample covariance matrix, where the threshold is selected using the slope heuristic. Based on the block-diagonal structure of the covariancematrix, the estimation problem is divided into several independent problems: subsequently, the network of dependencies between variables is inferred using the graphical lasso algorithm in each block. The performance of the procedure is illustrated on simulated data. An application to a real gene expression dataset with a limited sample size is also presented: the dimension reduction allows attention to be objectively focused on interactions among smaller subsets of genes, leading to a more parsimonious and interpretable modular network. Supplementary materials for this article are available online.</p>
                                                    </div>
                                                </div>
                                                <div class="item mix jpaper" data-year="2015">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                            <a href="https://tel.archives-ouvertes.fr/tel-01424124/" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>
                                                            <a href="https://tel.archives-ouvertes.fr/tel-01424124/document" class="tooltips" title="Download" target="_blank">
                                                                <i class="fa fa-cloud-download"></i>
                                                            </a>
                                                            
                                                        </div>


                                                        <h4 class="pubtitle">Classification et inférence de réseaux pour les données RNA-seq.</h4>
                                                        <div class="pubauthor"><strong>Gallopin M.</strong></div>
                                                        <div class="pubcite"><span class="label label-success">These</span> Manuscrit de thèse, NNT : 2015SACLS174, Université Paris Sud.</div>
                                                        
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>This thesis gathers methodologicals contributions to the statistical analysis of next-generation high-throughput transcriptome sequencing data (RNA-seq). RNA-seq data are discrete and the number of samples sequenced is usually small due to the cost of the technology. These two points are the main statistical challenges for modelling RNA-seq data. The first part of the thesis is dedicated to the co-expression analysis of RNA-seq data using model-based clustering. A natural model for discrete RNA-seq data is a Poisson mixture model. However, a Gaussian mixture model in conjunction with a simple transformation applied to the data is a reasonable alternative. We propose to compare the two alternatives using a data-driven criterion to select the model that best fits each dataset. In addition, we present a model selection criterion to take into account external gene annotations. This model selection criterion is not specific to RNA-seq data. It is useful in any co-expression analysis using model-based clustering designed to enrich functional annotation databases. The second part of the thesis is dedicated to network inference using graphical models. The aim of network inference is to detect relationships among genes based on their expression. We propose a network inference model based on a Poisson distribution taking into account the discrete nature and high inter sample variability of RNA-seq data. However, network inference methods require a large number of samples. For Gaussian graphical models, we propose a non-asymptotic approach to detect relevant subsets of genes based on a block-diagonale decomposition of the covariance matrix. This method is not specific to RNA-seq data and reduces the dimension of any network inference problem based on the Gaussian graphical model.In co-expression analyses of gene expression data, it is often of interest to interpret clusters of co-expressed genes with respect to a set of external information, such as a potentially incomplete list of functional properties for which a subset of genes may be annotated. Based on the framework of finite mixture models, we propose a model selection criterion that takes into account such external gene annotations, providing an efficient tool for selecting a relevant number of clusters and clustering model. This criterion, called the integrated completed annotated likelihood (ICAL), is defined by adding an entropy term to a penalized likelihood to measure the concordance between a clustering partition and the external annotation information. The ICAL leads to the choice of a model that is more easily interpretable with respect to the known functional gene annotations. We illustrate the interest of this model selection criterion in conjunction with Gaussian mixture models on simulated gene expression data and on real RNA-seq data.</p>
                                                    </div>
                                                </div>

                                                <div class="item mix jpaper" data-year="2015">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                            <a href="http://www.ncbi.nlm.nih.gov/pubmed/26461845" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>
                                                            <!--a href="http://faculty-gsb.stanford.edu/aaker/pages/documents/CultivatingAdmirationinBrands_JCP2012.pdf" class="tooltips" title="Download" target="_blank">
                                                                <i class="fa fa-cloud-download"></i>
                                                            </a-->
                                                            
                                                        </div>


                                                        <h4 class="pubtitle">A Model Selection Criterion For Model-based Clustering of Annotated Gene Expression Data.</h4>
                                                        <div class="pubauthor"><strong>Gallopin M.</strong>, Celeux G., Jaffrézic F.,  Rau A.</div>
                                                        <div class="pubcite"><span class="label label-success">Publication</span> Statistical Applications in Genetics and Molecular Biology, 2015, 14(5), 413-428.</div>
                                                        
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>In co-expression analyses of gene expression data, it is often of interest to interpret clusters of co-expressed genes with respect to a set of external information, such as a potentially incomplete list of functional properties for which a subset of genes may be annotated. Based on the framework of finite mixture models, we propose a model selection criterion that takes into account such external gene annotations, providing an efficient tool for selecting a relevant number of clusters and clustering model. This criterion, called the integrated completed annotated likelihood (ICAL), is defined by adding an entropy term to a penalized likelihood to measure the concordance between a clustering partition and the external annotation information. The ICAL leads to the choice of a model that is more easily interpretable with respect to the known functional gene annotations. We illustrate the interest of this model selection criterion in conjunction with Gaussian mixture models on simulated gene expression data and on real RNA-seq data.</p>
                                                    </div>
                                                </div>

                                                <div class="item mix jpaper" data-year="2013">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                            <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3740625/" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>
                                                            <!--a href="http://faculty-gsb.stanford.edu/aaker/pages/documents/CultivatingAdmirationinBrands_JCP2012.pdf" class="tooltips" title="Download" target="_blank">
                                                                <i class="fa fa-cloud-download"></i>
                                                            </a-->
                                                            
                                                        </div>

                                                        <h4 class="pubtitle">Data-based Filtering For Replicated High-throughput Transcriptome Sequencing Experiments.</h4>
                                                        <div class="pubauthor">Rau A., <strong>Gallopin M.</strong>, Celeux G., Jaffrézic F.</div>
                                                        <div class="pubcite"><span class="label label-success">Publication</span> Bioinformatics, 2013, 29(17)</div>
                                                        
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>Motivation: RNA sequencing is now widely performed to study differential expression among experimental conditions. As tests are performed on a large number of genes, stringent false-discovery rate control is required at the expense of detection power. Ad hoc filtering techniques are regularly used to moderate this correction by removing genes with low signal, with little attention paid to their impact on downstream analyses. Results: We propose a data-driven method based on the Jaccard similarity index to calculate a filtering threshold for replicated RNA sequencing data. In comparisons with alternative data filters regularly used in practice, we demonstrate the effectiveness of our proposed method to correctly filter lowly expressed genes, leading to increased detection power for moderately to highly expressed genes. Interestingly, this data-driven threshold varies among experiments, highlighting the interest of the method proposed here. Availability: The proposed filtering method is implemented in the R package HTSFilter available on Bioconductor.</p>
                                                    </div>
                                                </div>

                                                <div class="item mix jpaper" data-year="2013">
                                                    <div class="pubmain">
                                                        <div class="pubassets">
                                                            
                                                            <a href="#" class="pubcollapse">
                                                                <i class="fa fa-expand"></i>
                                                            </a>
                                                            <a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0077503" class="tooltips" title="External link" target="_blank">
                                                                <i class="fa fa-external-link"></i>
                                                            </a>
                                                            <!--a href="http://faculty-gsb.stanford.edu/aaker/pages/documents/CultivatingAdmirationinBrands_JCP2012.pdf" class="tooltips" title="Download" target="_blank">
                                                                <i class="fa fa-cloud-download"></i>
                                                            </a-->
                                                            
                                                        </div>

                                                        <h4 class="pubtitle">A Hierarchical Poisson Log-Normal Model for Network Inference from RNA Sequencing Data.</h4>
                                                        <div class="pubauthor"><strong>Gallopin M.</strong>, Rau A., Jaffrézic F.</div>
                                                        <div class="pubcite"><span class="label label-success">Publication</span> Plos ONE, 2013, 8(10)</div>
                                                        
                                                    </div>
                                                    <div class="pubdetails">
                                                        <h4>Abstract</h4>
                                                        <p>Gene network inference from transcriptomic data is an important methodological challenge and a key aspect of systems biology. Although several methods have been proposed to infer networks from microarray data, there is a need for inference methods able to model RNA-seq data, which are count-based and highly variable. In this work we propose a hierarchical Poisson log-normal model with a Lasso penalty to infer gene networks from RNA-seq data; this model has the advantage of directly modelling discrete data and accounting for inter-sample variance larger than the sample mean. Using real microRNA-seq data from breast cancer tumors and simulations, we compare this method to a regularized Gaussian graphical model on log-transformed data, and a Poisson log-linear graphical model with a Lasso penalty on power-transformed data. For data simulated with large inter-sample dispersion, the proposed model performs better than the other methods in terms of sensitivity, specificity and area under the ROC curve. These results show the necessity of methods specifically designed for gene network inference from RNA-seq data.</p>
                                                    </div>
                                                </div>

                                            </div>
                                        </div>
                                    </div>

                                </div>
                            </div>

                        </div>
                    </div>
                </div>

                
            </div>
        </div>
    </body>
</html>

